apps/core-app/src/shared/events/omni-panel.ts

```
@@ -1,4 +1,5 @@
 import { defineEvent, defineRawEvent } from '@talex-touch/utils/transport/event/builder'
+import type { DesktopContextCapsule } from '../intelligence/desktop-context-capsule'
```

ESLint

[error] Expected "../intelligence/desktop-context-capsule" (parent-type) to come before "@talex-touch/utils/transport/event/builder" (external).

(perfectionist/sort-imports)